libs/cardano-ledger-test/src/Test/Cardano/Ledger/Generic/Consensus.hs

```diff
@@ -1,1054 +0,0 @@
-{-# LANGUAGE AllowAmbiguousTypes #-}
-{-# LANGUAGE ConstraintKinds #-}
-{-# LANGUAGE DataKinds #-}
-{-# LANGUAGE FlexibleContexts #-}
-{-# LANGUAGE OverloadedStrings #-}
-{-# LANGUAGE RankNTypes #-}
-{-# LANGUAGE ScopedTypeVariables #-}
-{-# LANGUAGE TypeApplications #-}
-{-# LANGUAGE TypeFamilies #-}
-
--- | Generic Consensus examples
---   This module is meant to replace a bunch of Consensus examples
---   distributed across many Era. By defining the Consensus examples
---   for every era once, by writing code that is parameterized by a (Proof era),
---   it should be easier to maintain. It is meant to be byte for byte exact.
-module Test.Cardano.Ledger.Generic.Consensus where
```

Comment:
why did you remove these? I thought the plan was to switch the consensus example to use these.

Reply:
> I thought the plan was to switch the consensus example to use these.

Why would you wanna do that?

Reply:
the generic ones can be reused in future eras.

Reply:
If people stop coding with a copy+paste strategy then current consesus examples can also be reused in future eras. Otherwise there is nothing wrong with them. We have lenses for all our types which allow for much better and safer reusability of code, instead of the generic approach using `Proof`.

Another down side of using generic approach for consensus examples is we would have to release cardano-ledger-test to CHaPs and make this whole Proof thing into a public API. We don't want to do that.